### PR TITLE
reference-manual: boards: Added am64xx-sk flashing instructions

### PR DESCRIPTION
**Pull request by Tim-Anderson**
This provides the flashing instructions for the Texas Instruments AM64x SKEVM board.

Signed-off-by: Tim Anderson <tim.anderson@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
Docs for ccabee864d4a1b1434c5e5004caa098705d42272 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/891/docs/artifacts/html/index.html

**Review by vanmaegima — APPROVED** [review]

**Review by angolini — APPROVED** [review]

**Comment by doanac** [comment]
Docs for 5f064c62b88ef209938b9bb7282dcbdc60ed9fb6 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/893/docs/artifacts/html/index.html

**Review by munoz0raul — APPROVED** [review]
LGTM

**Comment by Tim-Anderson** [comment]
@MatthewCroughan can you please look at this and comment/approve? thanks in advance

File: source/reference-manual/boards/am64xx-sk.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+     Switch settings
+
+
+Flashing
+--------
+
+.. include:: generic-flashing.rst
+
+Remove the micro SDCard from the host and insert it into the socket on the bottom of the board.
```
Comment (MatthewCroughan) [inline source/reference-manual/boards/am64xx-sk.rst:33]:
```suggestion
1. Remove the micro SDCard from the host and insert it into the socket on the bottom of the board.
```

File: source/reference-manual/boards/am64xx-sk.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+.. figure:: /_static/boards/am64xx-sk-bottom.png
+     :width: 600
+     :align: center
+
+     SDCard location
+
+Power on the board to boot the new image.
```
Comment (MatthewCroughan) [inline source/reference-manual/boards/am64xx-sk.rst:41]:
```suggestion
2. Power on the board to boot the new image.
```

**Review by MatthewCroughan — CHANGES_REQUESTED** [review]

File: source/reference-manual/boards/am64xx-sk.rst (hunk 2 above)
Comment (Tim-Anderson) [reply source/reference-manual/boards/am64xx-sk.rst:41]: the step numbers are consistent with the other board installation instructions. Are you suggesting that all of them not have step numbers?

Comment (Tim-Anderson) [reply source/reference-manual/boards/am64xx-sk.rst]: done

**Comment by doanac** [comment]
Docs for a5f59fa7b69393aea84418bf34463c06988e14c3 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/896/docs/artifacts/html/index.html

**Pull request merged**